Update readme

dginelli opened the pull request:
Updated the README.md file with the information about the data contained in the repository.

dginelli committed:
Added information about the builds in the README

dginelli committed:
Added information about the build time

dginelli committed:
Added the information about the different types of the repairnator files

dginelli committed:
Divided the repairnator files by type and added the information about the bug type

dginelli committed:
Added the information about the most common failure types

dginelli committed:
Added information about the number of the builds with a certain number of failing/erroring test cases

dginelli committed:
Updated the general information of the README.md

dginelli committed:
Updated the section "Builds Information extracted from the Travis CI API" of the README.md

dginelli committed:
Updated the section Builds Information extracted from the Travis CI API in the README.md

dginelli committed:
Added information about the repairnator.json file in README.md

dginelli committed:
Added information about the Repairnator files in README.md

dginelli committed:
Updated the most common test failure types in the README.md

dginelli committed:
Updated the information about the failing and erroring test cases in README.md

dginelli committed:
Updated the section "Builds information extracted using Repairnator" in README.md

dginelli committed:
Updated README.md

Added the introduction for the section "Builds information extracted from the Travis CI API".

dginelli committed:
Updated the README.md with minor changes

dginelli committed:
Updated README.md

Added the information about the content of the Repairnator files

dginelli committed:
Updated README.md

Fixed the Bibtex element

dginelli committed:
Updated README.md

Minor changes and corrected a wrong number in the section related to the most common test failure types.

monperrus committed:
Update README.md

monperrus commented on README.md:
what is the corresponding exception?

dginelli replied:
This was indeed a point that I wanted to write in the previous email, but I forgot to cite it. Looking at the repairnator.json files, there are some of them in which there is the voice "skipped" as one of the values of the array "FailureNames" contained in the object "metrics". This is an example: https://github.com/repairnator/repairnator-experiments/blob/70a4fbb5517facbcc39621a3ca5fd79256c246ad/repairnator.json#L9

As you can see, there is the voice “java.lang.AssertionError” together with the voice “skipped”, but there are no other details about the type of the exception. Thus, I don’t know if this voice is associated with a specific exception or not.

monperrus commented on README.md:
it's never 0?

dginelli replied:
Yes, there are also builds with 0 failing test cases (they are 5.024) and builds with 0 erroring test cases (they are 7.085), but I have not put these values in the two tables.

dginelli committed:
Updated README.md

Added the note about the fact that there are some repairnator.json files in which the exception name has been saved both with the correct name and with a single ":" at the end of the name.

monperrus commented:
OK, thanks. Could you put the values in both tables?

monperrus commented:
For this, we would have to look in the code of Repairnator to see what it means.

dginelli committed:
Update README

Updated the numbers in README.md after deleting build JSON files and corresponding branches associated with the builds collected in 2016, October 2018 and April 2019.

dginelli committed:
Update README.md

Removed the failure name "skipped" from the table and added another one.

dginelli committed:
Update README.md

Added information about the jobs associated with the collected builds.

dginelli committed:
Update README.md

Changed the order of examples in the section "Number of failed jobs per build".

monperrus commented:
ready to merge?

dginelli commented:
> ready to merge?

If we don't have to put the data about Maven modules and pom.xml files., I think the pull request can be merged.

monperrus commented:
Thanks a lot! We'll continue to evolve this page in subsequent PRs :-)
